modules/twinklebatchdelete.js

```diff
@@ -55,7 +55,7 @@ Twinkle.batchdelete.callback = function twinklebatchdeleteCallback() {
 							checked: true
 						},
 						{
-							label: 'Delete redirects to deleted pages',
+							label: 'Delete redirects to deleted pages (and delete the redirect talk pages)',
```

Comment:
While I agree a separate checkbox for redirect talk pages is unnecessary, maybe it would be better to delete them only if delete_talk above is checked?